cmd/zpool/zpool_main.c

```diff
@@ -31,6 +31,7 @@
  * Copyright (c) 2017 Open-E, Inc. All Rights Reserved.
  * Copyright (c) 2017, Intel Corporation.
  * Copyright (c) 2019, loli10K <ezomori.nozomu@gmail.com>
+ * Portions Copyright (c) 2021, Colm Buckley <colm@tuatha.org>
```

Comment:
I don’t think “Portions” is necessary. It’s a bit of a different question if that should be removed from existing copyright lines, but I wouldn’t add more.

Reply:
Hmm. The instructions in the CDDL header do say to use "Portions" but almost nobody else has done so. Consistency would be good; anyone care to ask a lawyer?

Reply:
@behlendorf Would you like to weigh in here? I don't have strong feelings either way.

Reply:
Since `git` does a much better job of it our policy has always been to rely on the commit history for proper attribution.  However, we do maintain the copyright headers as a courtesy to for all the contributors.  Normally we opt for consistency here so I'd suggest dropping the "Portions" bit too.